libs/ui-watt/src/lib/foundations/color/_colors.import.scss

```diff
@@ -0,0 +1,89 @@
+@use 'sass:map';
+
+/*
+ * @Please notice!!! 
+ * These values should be kept in-sync with colors.ts
+ * Naming convention is as follow: --watt-color-{COLOR GROUP (ONLY IF NOT THE SAME AS COLOR NAME)}-{COLOR NAME}
```

Comment:
Suggestion: Consider removing this line because it sounds confusing having a conditional in it.